CHANGELOG.md

```diff
@@ -1,16 +1,39 @@
 ## 0.17.0 (unreleased)
 
+This release removes all methods declared as _deprecated_ in the 0.16 release, experimental Selenium support and much more! If you are looking to upgrade from an earlier release, it is recommended to first go to 0.16.x.
+Other goodies include improved test helpers, a cookies API and handling for JS-dialogues.
+
+
+### Breaking Changes
```

Comment:
I added `Breaking Changes` because imo these are neither Features nor Bugfixes and often also the most important things when I look at a Changelog